### PR TITLE
[4.0] Remove horizontal scrollbar in statistics modal

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
Use `container-popup` to remove extra margins resulting in horizontal scrollbar. Make markup consistent with other modals.

### Testing Instructions
Go to Components > Smart Search
Click `Statistics` button
See horizontal scrollbar
Apply PR
No horizontal scrollbar

### Actual Results

statistics-popup

### CODE REVIEW

File: administrator/components/com_finder/tmpl/statistics/default.php [hunk 1]
```diff
@@ -12,44 +12,42 @@
...
-					<td>
-						<span class="badge badge-info"><?php echo number_format($this->data->link_count, 0, Text::_('DECIMALS_SEPARATOR'), Text::_('THOUSANDS_SEPARATOR')); ?></span>
-					</td>
-				</tr>
-			</tbody>
-		</table>
-	</div>
+<div class="container-popup">
+	<p class="tab-description"><?php echo Text::sprintf('COM_FINDER_STATISTICS_STATS_DESCRIPTION', number_format($this->data->term_count, 0, Text::_('DECIMALS_SEPARATOR'), Text::_('THOUSANDS_SEPARATOR')), number_format($this->data->link_count, 0, Text::_('DECIMALS_SEPARATOR'), Text::_('THOUSANDS_SEPARATOR')), number_format($this->data->taxonomy_node_count, 0, Text::_('DECIMALS_SEPARATOR'), Text::_('THOUSANDS_SEPARATOR')), number_format($this->data->taxonomy_branch_count, 0, Text::_('DECIMALS_SEPARATOR'), Text::_('THOUSANDS_SEPARATOR'))); ?></p>
+	<table class="table table-sm">
```
Comment (brianteeman) [inline administrator/components/com_finder/tmpl/statistics/default.php:17]: table-sm does not exist

Comment (Quy) [reply administrator/components/com_finder/tmpl/statistics/default.php:17]: It is there in template.css.

```
.table-sm th,
.table-sm td {
  padding: 0.3rem; }
```

Comment (brianteeman) [reply administrator/components/com_finder/tmpl/statistics/default.php:17]: no idea why my search didnt find it

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 7938fb2f0bc1e53bb50bdf9e8873f800a775e353<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26842">issues.joomla.org/tracker/joomla-cms/26842</a>.</sub>

**Comment by infograf768** [comment]
Note: 
1. we use `container-popup` in many places in core but this class does not exist in css.
2. The table has no left border. It does not show much when used in the full page but it shows in such popups as background is white.
To solve this I added a border in `/administrator/templates/atum/scss/vendor/bootstrap/_table.scss`
```
.table {
  box-shadow: 1px 1px 1px 0px rgba(0, 0, 0, 0.25);
  border: 1px solid var(--atum-bg-light); // added
```
What do you think?

**Comment by Quy** [comment]
Within the module and modal, I prefer no right and left borders.

**Comment by DerTebbe** [comment]
I have tested this item :white_check_mark: successfully on fd52ada69adf6c0875f47975977f82b310a5735c<br /><br />Statistics modal without horizontal scrollbar
Tested successfully <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26842">issues.joomla.org/tracker/joomla-cms/26842</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26842">issues.joomla.org/tracker/joomla-cms/26842</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**